Enterprise-CMCS/managed-care-review
# MCR-4255: do not display NEW tag on initial submissions

## Summary

This is small PR for ensuring that the 'NEW' tag doesn't appear on the submission summary for initial submissions

This also fixes an underlying bug we had with 'date added' being displayed on initial submission

#### Related issues
https://jiraent.cms.gov/browse/MCR-4255

#### Screenshots

Initial submission (no new tag)
<img width="861" alt="Screenshot 2024-07-09 at 4 37 16 PM" src="https://github.com/Enterprise-CMCS/managed-care-review/assets/67110378/b302feb9-9320-4bbc-be7f-1eb8e961e261">

After unlock and resubmitting with additional document (new tag present)
<img width="878" alt="Screenshot 2024-07-09 at 4 38 11 PM" src="https://github.com/Enterprise-CMCS/managed-care-review/assets/67110378/d0a2b044-5cd5-43d4-8ca6-2879a6590e72">


#### Test cases covered

- RateDetailsSummarySection - NEW tag not displayed on initial submission
- ContractDetailsSummarySection - NEW tag not displayed on initial submission

## QA guidance

- Create a new contract and rate submission 
- Upload documents and fill out details and submit 
- Log in as a CMS user and view submission just made
- Notice on the submission summary page none of the documents on the initial submission are marked as "new"
